charts/gafaelfawr/templates/configmap.yaml

```diff
@@ -6,46 +6,46 @@ metadata:
     {{- include "gafaelfawr.labels" . | nindent 4 }}
 data:
   gafaelfawr.yaml: |
-    realm: {{ .Values.host | quote }}
-    loglevel: {{ .Values.loglevel | quote }}
+    realm: {{ required "config.host must be set" .Values.config.host | quote }}
```

Comment:
is that like an error string? ("config.host must be set")

Reply:
Yup, exactly. `required` lets you determine at install time that a required values parameter was missing and produce a reasonable error message instead of some bizarre failure later in the system.

Reply:
Nice that is super useful.  Sometimes it doesn't make sense to have a default in the chart or values file, but no way to alert to it.  This seems perfect for that